### PR TITLE
Add input validation for overall wall time limit

### PR DESCRIPTION
**Pull request by iqbalcodes6602**
# Descripción

I have made a functionality to update the value of that input field whenever the user enters the value greater than 60000 in the front-end. The vlaue in the input field will be automatically turned to 60000 if any vlaue entered is greater than 60000, thus not allowing user user to enter any value greater.

Fixes: #7307

# Comentarios

Si hay comentarios para los reviewers (por ejemplo, dónde hay que empezar a
revisar, algo a lo que hay que poner atención adicional, etc.), todo eso se
puede poner en esta sección.

También si faltan cosas por hacer, se puede hacer un checklist en esta sección.

Si no se utiliza, esta sección se puede eliminar.

# Checklist:

- [x] El código sigue la [guía de
      estilo](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) de
      omegaUp.
- [x] Se corrieron todas las pruebas y pasaron.
- [x] Si se está agregando funcionalidad nueva, se agregaron pruebas.
- [x] Si el cambio es grande (> 200 líneas), hay que intentar partirlo en
      varios pull requests. De preferencia uno para los controladores + phpunit
      y luego otro para la interfaz.

### CODE REVIEW

**Comment by iqbalcodes6602** [comment]
this is the changes i have made (ignore the text above the input field).

https://github.com/omegaup/omegaup/assets/111554475/13c8c176-5efa-4caa-b20d-3a1bf6b7ccda

File: frontend/www/js/omegaup/components/problem/Settings.vue [hunk 1]
```diff
@@ -124,5 +125,13 @@ export default class Settings extends Vue {
   @Prop() validator!: string;
 
   T = T;
+
+  validateOverallWallTimeLimitInput(event: any) {
+    if (Number(event.target.value) > 60000) {
+      this.overallWallTimeLimit = 60000;
+      const overallWallTimeLimitInput = document.getElementsByName('overall_wall_time_limit')[0] as HTMLInputElement;
```
Comment (pabo99) [inline frontend/www/js/omegaup/components/problem/Settings.vue:132]: Is there any way to solve this using `Vue.js`. A [`Watcher`](https://v2.vuejs.org/v2/guide/computed.html#Watchers) could be a choice. You may need to update the directive `v-bind` with `v-model`.

**Review by pabo99 — COMMENTED** [review]
Just one comment

File: frontend/www/js/omegaup/components/problem/Settings.vue (hunk 1 above)
Comment (iqbalcodes6602) [reply frontend/www/js/omegaup/components/problem/Settings.vue:132]: ok will try

Comment (iqbalcodes6602) [reply frontend/www/js/omegaup/components/problem/Settings.vue]: hello @pabo99 i have added another commit with the watcher as  you said and its working correctly, can you please check.

**Comment by iqbalcodes6602** [comment]
hello @pabo99 i have added another commit with the watcher as you said and its working correctly, can you please check.

File: frontend/www/js/omegaup/components/problem/Settings.vue [hunk 2]
```diff
@@ -38,15 +38,14 @@
           T.problemEditFormWallTimeLimit
         }}</label>
         <input
+          v-model="localOverallWallTimeLimit"
```
Comment (pabo99) [inline frontend/www/js/omegaup/components/problem/Settings.vue:41]: Just for convention:
```suggestion
          v-model="currentOverallWallTimeLimit"
```
We commonly use the prefix `current` when we are going to modify the value of a property.

File: frontend/www/js/omegaup/components/problem/Settings.vue [hunk 3]
```diff
@@ -126,11 +125,11 @@ export default class Settings extends Vue {
...
-  validateOverallWallTimeLimitInput(event: any) {
-    if (Number(event.target.value) > 60000) {
-      this.overallWallTimeLimit = 60000;
-      const overallWallTimeLimitInput = document.getElementsByName('overall_wall_time_limit')[0] as HTMLInputElement;
-      overallWallTimeLimitInput.value = '60000';
+  localOverallWallTimeLimit = this.overallWallTimeLimit;
+  @Watch('localOverallWallTimeLimit')
+  onOverallWallTimeLimitChanged(newVal: number) {
+    if (newVal > 60000) {
+      this.localOverallWallTimeLimit = 60000;
```
Comment (pabo99) [inline frontend/www/js/omegaup/components/problem/Settings.vue:132]: nit:
```suggestion
  localOverallWallTimeLimit = this.overallWallTimeLimit;
  
  @Watch('localOverallWallTimeLimit')
  onOverallWallTimeLimitChanged(newVal: number) {
    if (newVal > 60000) {
      this.localOverallWallTimeLimit = 60000;
```

**Review by pabo99 — COMMENTED** [review]
Excellent!

This PR looks much better now! I've left you a couple of suggestions, and once you apply the corresponding changes, I can approve the PR.

Another thing: Could you please upload a video (a GIF is okay) in the description of the PR? This way, we'll know exactly how your changes are improving the UI/UX.

Thank you for your help.

**Comment by iqbalcodes6602** [comment]
hello @pabo99 finally i have done everything as you said, can you please approve the PR. 
I have changed localOverallWallTimeLimit to currentOverallWallTimeLimit and added a line break as well.

**Review by pabo99 — APPROVED** [review]
LGTM

Thank you for solving this issue!

**Comment by iqbalcodes6602** [comment]
> LGTM
> 
> Thank you for solving this issue!

when will this change be merged?

**Comment by iqbalcodes6602** [comment]
hello @heduenas can you please merge this PR since it has been already approved

**Comment by iqbalcodes6602** [comment]
Hello @pabo99 sir, 
These changes which are approved by you, when will they be merged?

**Comment by pabo99** [comment]
> Hello @pabo99 sir, These changes which are approved by you, when will they be merged?

Hi @iqbalcodes6602 , the changes will be merged soon, just be patient! :)

**Review by heduenas — APPROVED** [review]

**Pull request merged**